src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,26 @@
+
+$cardwidth: 200px;
+$gapX: 48px;
+$gapY: 46px;
+$cardCountInRow: 4;
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  width: 1024px;
```

Comment:
And write function which will calculate max-width for catalog (rule 13 in checklist: [LAYOUT] - Use sass functions for dynamic max-width calculation, your catalog container should work fine for any number of cards.)

Reply:
Fix your mate's comment about function